cmake/Config.cmake.in

```diff
@@ -0,0 +1,16 @@
+set(IMGUI_VERSION "@IMGUI_VERSION@")
```

Comment:
I think this one not needed since version will be saved in the file generated by `write_basic_package_version_file`.